src/main/resources/mybatis/collections.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I found this file very hard to read. Both the formatting and the number of chunks (only used in one place).